### PR TITLE
Update legacy terminology in a few places

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , can you review this when you get a chance? Low priority

File: cts/scheduler/xml/cancel-behind-moving-remote.xml [hunk 1]
```diff
@@ -359,19 +359,19 @@
         <primitive class="ocf" id="ovndb_servers" provider="ovn" type="ovndb-servers">
           <instance_attributes id="ovndb_servers-instance_attributes">
             <nvpair id="ovndb_servers-instance_attributes-inactive_probe_interval" name="inactive_probe_interval" value="180000"/>
-            <nvpair id="ovndb_servers-instance_attributes-listen_on_master_ip_only" name="listen_on_master_ip_only" value="yes"/>
+            <nvpair id="ovndb_servers-instance_attributes-listen_on_promoted_ip_only" name="listen_on_master_ip_only" value="yes"/>
```
Comment (nrwahl2) [inline cts/scheduler/xml/cancel-behind-moving-remote.xml:362]: still got a master in there; necessary?

File: cts/scheduler/xml/cancel-behind-moving-remote.xml [hunk 2]
```diff
@@ -359,19 +359,19 @@
...
             <nvpair id="ovndb_servers-instance_attributes-manage_northd" name="manage_northd" value="yes"/>
-            <nvpair id="ovndb_servers-instance_attributes-master_ip" name="master_ip" value="172.17.1.87"/>
-            <nvpair id="ovndb_servers-instance_attributes-nb_master_port" name="nb_master_port" value="6641"/>
-            <nvpair id="ovndb_servers-instance_attributes-nb_master_protocol" name="nb_master_protocol" value="ssl"/>
+            <nvpair id="ovndb_servers-instance_attributes-promoted_ip" name="master_ip" value="172.17.1.87"/>
+            <nvpair id="ovndb_servers-instance_attributes-nb_promoted_port" name="nb_master_port" value="6641"/>
+            <nvpair id="ovndb_servers-instance_attributes-nb_promoted_protocol" name="nb_master_protocol" value="ssl"/>
```
Comment (nrwahl2) [inline cts/scheduler/xml/cancel-behind-moving-remote.xml:366]: here too; these might change the outputs, not sure, but worth fixing in this PR?

Comment (nrwahl2) [reply cts/scheduler/xml/cancel-behind-moving-remote.xml:366]: Probably would be an ordeal so never mind; I'm realizing there are a lot of names and such scattered around with it

**Comment by nrwahl2** [comment]
Looks good to me.

**Pull request merged**